### ISSUE TITLE
Labelling after superpixels first appear doesn't always get recorded

### ISSUE THREAD

**Issue by manthey** [DigitalSlideArchive/wsi-superpixel-guided-labeling#124]
I was using a small dataset (just one image: TCGA-DQ-5630-01Z-00-DX1.07FE0581-2412-43DA-96A9-0DA192DAED3D.svs) and started a new project.  After the superpixels appear, I created three labels and clicked some appropriate superpixels for each label, but then training fails because the superpixels are not actually labelled.  The UI shows that I labelled a few of each category, but reloading shows zero for each.  Looking at the rest requests to store the new labels, it looks like the vector is all zeros, like the new categories are all zero not 1, 2, 3.

This happens on the master branch and on the toggle-modes branch.

**Comment by bnmajor** [comment]
Resolved in [8700ea3](https://github.com/DigitalSlideArchive/wsi-superpixel-guided-labeling/pull/120/commits/8700ea3c7c841e8c9ae546f2290f93f8389a6664).

This issue can be closed once #120 is merged into master.
